frontend/packages/console-app/src/components/quick-starts/utils/quick-start-context.tsx

```diff
@@ -174,7 +175,7 @@ export const useValuesForQuickStartContext = (): QuickStartContextValues => {
     [activeQuickStartID, setAllQuickStartStates, fireTelemetryEvent],
   );
 
-  const language = localStorage.getItem('bridge/language') || 'en';
+  const language = getLastLanguage() || 'en';
```

Comment:
Not new, but `en` is not the right default. Can you open a separate bug to look at this?

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=2001125